chipy_org/apps/meetings/serializers.py

```diff
@@ -4,7 +4,7 @@
 
 
 class PresentorSerializer(serializers.ModelSerializer):
-    email = serializers.SerializerMethodField('get_email')
```

Comment:
Rest framework was throwing this error when I included that method call.  Removing the call seemed to fix it and I think I see the appropriate email address in the meetings list api view.  Maybe this is a change in rest framework 3?  

    AssertionError at /api/meetings/
    It is redundant to specify `get_email` on SerializerMethodField 'email' in serializer 'PresentorSerializer', because it is the same as the default method name. Remove the `method_name` argument.